File: docs/src/pages/GuidePage/content/validation/code/validate_yaml.py
from lcax import ValidationSchema, validate, Project
from pathlib import Path
import yaml

project = Project.loads(Path("project.json").read_text())
schemas = [
    ValidationSchema(**schema) for schema
    in yaml.load(Path("validation_rules.yaml").read_text())
]

result = validate(project, schemas) # result = []

File: docs/src/pages/GuidePage/content/validation/code/validate_yaml.rs
use std::fs;
use std::path::Path;

fn main() {
    let project = serde_json::from_str::<Project>(
        &fs::read_to_string("project.json").unwrap()
    ).unwrap();

    let schemas = serde_yml::from_str::<Vec<ValidationSchema>>(
        &fs::read_to_string("validation_schema").unwrap()
    ).unwrap();

    let result = validate(&project, &schemas); // result = []
}